roles/debian_hypervisor/tasks/main.yml

```
@@ -210,6 +210,7 @@
     group: root
     mode: 0644
   register: ovsvswitchd
+  when: team0_0 is defined
```

Is it possible to use `host: cluster_machines` ([ansible pattern](https://docs.ansible.com/ansible/latest/inventory_guide/intro_patterns.html)) ?

I'm afraid that testing team0_0 definition might be too fragile.

---

I don't think it might be better as the task is called within a role, for host `standalone_machine` and `hypervisors`, and so must be compatible even if hypervisors host is in `cluster_machines` or `standalone_machine` groups.
`team0_0` is a documented var is the example inventory and SEAPATH documentation, and used when we setup a cluster.
Otherwise another solution would be to move the tasks added with https://github.com/seapath/ansible/pull/666 in a cluster role, such as `roles/network_clusternetwork/tasks/main.yml`

---

>  team0_0 is a documented var is the example inventory and SEAPATH documentation, and used when we setup a cluster.

Yes, you are right. Ok, I'm fine with merging with the team0_0 test, but should it be
```suggestion
  when: team0_0 is defined and team0_1 is defined
```
? (since both variables are used in template)